app/graphql/types/wallets/recurring_transaction_rules/update_input.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Types
+  module Wallets
+    module RecurringTransactionRules
+      class UpdateInput < Types::BaseInputObject
+        graphql_name 'UpdateRecurringTransactionRuleInput'
+
+        argument :granted_credits, String, required: true
+        argument :lago_id, ID, required: false
+        argument :interval, Types::Wallets::RecurringTransactionRules::IntervalEnum, required: false
```

Comment:
GraphQL/OrderedArguments: Arguments should be sorted in an alphabetical order within their section. Field `interval` should appear before `lago_id`.